manager_self_clearing.go

```diff
@@ -42,6 +43,7 @@ func NewSelfClearingProcessManager(maxProcs int, trackProcs bool) (Manager, erro
 //
 // The self clearing process manager is not thread safe. Wrap with the
 // local process manager for multithreaded use.
+// TODO: allow local process manager to wrap other managers.
```

Comment:
can we do this? or open a ticket rather than leaving a TODO in the code?

Reply:
I added the ticket number to the TODO. It would require some changing of local manager internals, since it currently assumes  that it's wrapping a basic manager.